### PR TITLE
adding flux config for staging-lg mirrornode deployment

### PR DESCRIPTION
**Pull request by tolorukooba**
Signed-off-by: tolorukooba <teslim.olorukooba@swirldslabs.com>

This PR adds flux config for staging-lg mirrornode kube deployment.

### CODE REVIEW

**Review by edwin-greene — APPROVED** [review]
Looks good to me.

File: clusters/preprod/staging-lg/sealed-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...JcDOqVdyfLdqCHMtFWYnyD/Oolu3O9YL//1MNHSDjG+WYmbuc1F7DG4PT2kinKI56FR+I3sk4BvXnA1BsaXoyykFxnslZT5lqgOTjok2BFjkeOzCIjZK3J7zMGSWrrzVF8rHdzmqlh+L4Ry1YrUpGDionB2u4XIm+OBcxKDPxGvIsPe01IJk8eQi9QMni4um9FvPhCdRd7dvD4axdby3SxnWjK+o+iC2hjzjIu22jh+pI99X6xm9nAb9n6EedeGOg7nSd9uAaEWnI1KhNFyk4HDX2rYmy2ogMJO2BQ9PyZ/B+sj6ovQuLYgXD1KPnZm7Bs2LRnfMX/VWJQXvt9uEt0U2uumjRE5+Oxk3di69IlfsuvxPxL5JDD3qvv/LfQONb5XXasCgdrTK8TE27/nJqkP5Jarc+YJX/ybkiwOyAly2Laid7f1AUM0vhmJ2pQrFuxoNq9FKtBtpHtvMJgL2jPgHQBWCW7msacmd6xyKA/Zip/oHnOD3+kEI2WZOo162hu5yVFbZYVVKmmvZiOhXMR51umCIPqJu22lsYnTTdjRVOcXGBe69aU+bSZhZegaSleq0hV6ByW3XufBXJBaVhFElNRmtbCx3BoUspRxbFLt8Bb0x+UVRWWbcIIGqhRUFoO0+3klZXdHgpsM8M5KCtmw2nFbQBj+6z2DW/QBi/UJIMHVNg/QDZ2aDeGklmeKYHWL4JO13d0Yqo8mILLDFOfw4is+CCdMVptWTt3ZeFuQj/abDTNGKH7gHz/jacElEA1X+Fkj5Aeu878795+rH63N4J+JM1hLXGzcgBEyLf57Uet4KsuC2IHw37kDbmy9ZjvQYCmZX2HAM3IzJHFjv5O2yoCRC72nZBb/EHufuadGXKP/fj54jS+5JbyRFF8x6KMIM60qy6jLGblDtCKPIYSf2sL0C/7SJ5mTuJ+RaBZkB6yTgXBewyAkp6la9uvwGINoRtQ2KC2Hpd+4qCCX3J/IAsWTdcmeJIao4J9+EYz5gS6jfTfShN8IskrjHFY+lwjYkhdHFbeaFL4xONntWT9Np3axMfYmGfyQaC7hG17Qr4EFT62Yc+VpFyCWTDGfiF4faQ0CEGwxCDiS+MMLvCro+em4pv9fozxpY46R6s7qoqzjcPIEwwkXg/+KVhhR+ZuPYvzpk6ykDbeklHU7/Mmx9D4QmPxwMC7miUcpb7sxamslS
```
Comment (steven-sheehy) [inline clusters/preprod/staging-lg/sealed-secret.yaml:9]: The bucket name should have an `s` at the end.

Comment (steven-sheehy) [inline clusters/preprod/staging-lg/sealed-secret.yaml:9]: Need to run init SQL and create users and updated secret.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Review by mgoelswirlds — APPROVED** [review]

File: clusters/preprod/staging-lg/sealed-secret.yaml (hunk 1 above)
Comment (tolorukooba) [reply clusters/preprod/staging-lg/sealed-secret.yaml]: @steven-sheehy fixed!

Comment (tolorukooba) [reply clusters/preprod/staging-lg/sealed-secret.yaml]: done!

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**